quipucords/scanner/network/processing/util.py

```diff
@@ -178,3 +178,58 @@ def process(output):
         """
         return {item['item']: item['stdout'] if item['rc'] == 0 else False
                 for item in output['results']}
+
+
+class FuseVersionProcessor(process.Processor):
```

Comment:
Is there use of this outside of `karaf.py`? Wondering if this is either to specifically named or shouldn't be in the `utils.py`.